packages/form-js-viewer/test/spec/render/components/form-fields/util/valuesUtil.spec.js

```diff
@@ -0,0 +1,180 @@
+import { normalizeValuesData } from '../../../../../../src/render/components/util/valuesUtil.js';
+
+describe('valuesUtil', function() {
+
+  describe('#normalizeValuesData', function() {
+
+    it('should not alter fully defined value', function() {
+
+      // given
+      const values = [ { value: 'john', label: 'John' }, { value: 'jessica', label: 'Jessica' } ];
+
+      // when
+      const result = normalizeValuesData(values);
+
+      // then
+      expect(result).to.eql([ { value: 'john', label: 'John' }, { value: 'jessica', label: 'Jessica' } ]);
+    });
+
+
+    it('should filter out null', function() {
+
+      // given
+      const values = [ { value: 'john', label: 'John' }, { value: 'jessica', label: 'Jessica' }, null ];
+
+      // when
+      const result = normalizeValuesData(values);
+
+      // then
+      expect(result).to.eql([ { value: 'john', label: 'John' }, { value: 'jessica', label: 'Jessica' } ]);
+    });
+
+
+    it('should filter out undefined', function() {
+
+      // given
+      const values = [ { value: 'john', label: 'John' }, { value: 'jessica', label: 'Jessica' }, undefined ];
+
+      // when
+      const result = normalizeValuesData(values);
+
+      // then
+      expect(result).to.eql([ { value: 'john', label: 'John' }, { value: 'jessica', label: 'Jessica' } ]);
+    });
+
+
+    it('should add label if not provided', function() {
+
+      // given
+      const values = [ { value: 'john' }, { value: 'jessica' } ];
+
+      // when
+      const result = normalizeValuesData(values);
+
+      // then
+      expect(result).to.eql([ { value: 'john', label: 'john' }, { value: 'jessica', label: 'jessica' } ]);
+    });
+
+
+    it('should ignore valuesData without value', function() {
+
+      // given
+      const valuesData = [ { label: 'John' }, { label: 'Jessica' } ];
+
+      // when
+      const result = normalizeValuesData(valuesData);
+
+      // then
+      expect(result).to.eql([]);
+    });
+
+
+    it('should convert string definitions to value/label objects', function() {
+
+      // given
+      const valuesData = [ 'john', 'jessica' ];
+
+      // when
+      const result = normalizeValuesData(valuesData);
+
+      // then
+      expect(result).to.eql([ { value: 'john', label: 'john' }, { value: 'jessica', label: 'jessica' } ]);
+    });
+
+
+    it('should filter out random objects', function() {
```

Comment:
```suggestion
    it('should filter out invalid objects', function() {
```

Just to prevent we do not know what random objects mean once we have a look at this time again 🙂